packages/renderer/src/lib/image/BuildImageFromContainerfile.svelte

```diff
@@ -328,6 +282,55 @@ async function abortBuild(): Promise<void> {
   buildImageInfo.buildRunning = false;
   buildImageInfo.buildFinished = true;
 }
+
+let platforms = $derived(buildImageInfo.containerBuildPlatform ? buildImageInfo.containerBuildPlatform.split(',') : []);
+let containerFilePath = $derived(buildImageInfo.containerFilePath);
```

Comment:
question: why declaring a new derived state? `buildImageInfo` is already a state, so, using `buildImageInfo.containerFilePath` in other derived state should work directly?

Reply:
Because the migation script did so but I will simplify thanks :)